torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -270,6 +270,57 @@ RegisterOperators reg({
             return 0;
           };
         }),
+    Operator(
```

Comment:
`prim::ConstantChunk` and `prim::FusedChunk` are exactly the same right? Can we just keep one or the other? It is confusing to have two.

Reply:
They are the same. I kept them both because it separates the concerns of the fuser a bit better (we can only have pointwise ops and `prim::Fused*` inside our kernels). I can deduplicate them if you really want to.